packages/typechain/src/parser/normalizeName.ts

```diff
@@ -1,16 +1,40 @@
-import { upperFirst, camelCase } from 'lodash'
+import { upperFirst } from 'lodash'
 
 /**
  * Converts valid file names to valid javascript symbols and does best effort to make them readable. Example: ds-token.test becomes DsTokenTest
  */
 export function normalizeName(rawName: string): string {
-  const t1 = rawName.split(' ').join('-') // spaces to - so later we can automatically convert them
-  const t2 = t1.replace(/^\d+/, '') // removes leading digits
-  const result = upperFirst(camelCase(t2))
+  const transformations: ((s: string) => string)[] = [
+    (s) => s.split(' ').join('-'), // spaces to - so later we can automatically convert them
```

Comment:
isn't it prettier with a replace? :smiley: also takes care of sequences of multiple and different whitespace chars
```suggestion
    (s) => s.replace(/\s+/g, '-'), // spaces to - so later we can automatically convert them
```